.erb/scripts/electron-rebuild.js

```diff
@@ -1,13 +1,12 @@
 import path from 'path';
 import { execSync } from 'child_process';
 import fs from 'fs';
-import { dependencies } from '../../src/package.json';
-
-const nodeModulesPath = path.join(__dirname, '../../src/node_modules');
+import { dependencies } from '../../build/app/package.json';
+import webpackPaths from '../configs/webpack.paths.js';
 
 if (
   Object.keys(dependencies || {}).length > 0 &&
-  fs.existsSync(nodeModulesPath)
+  fs.existsSync(webpackPaths.nodeModulesPath)
```

Comment:
This should be `appNodeModulesPath`. It takes hours to find why native build not work.